### PR TITLE
Remove internal feed

### PR DESCRIPTION
**Pull request by ViktorHofer**
I triggered an internal build and the packages are now in the dotnet-core feed.

### CODE REVIEW

**Comment by ViktorHofer** [comment]
Not sure what's going on here...

File: global.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
         "version": "2.1.300"
     },
     "msbuild-sdks": {
-        "Microsoft.DotNet.Arcade.Sdk": "1.0.0-bootstrap-1"
+        "Microsoft.DotNet.Arcade.Sdk": "1.0.0-prerelease-63123-07"
```
Comment (weshaggard) [inline global.json]: Looks like the version you want is `prerelease-63123-27`

Comment (ViktorHofer) [reply global.json]: Where did you get that number from?..

**Comment by ViktorHofer** [comment]
@dotnet-bot test this please

File: global.json (hunk 1 above)
Comment (weshaggard) [reply global.json]: From the vsts build that kicked off after the earlier change.

**Comment by ViktorHofer** [comment]
> I'd wait with this - I had to fix up the signing logic. Will need to test it.

@tmat can we remove the feed now?

**Comment by maririos** [comment]
@ViktorHofer this PR is no longer needed. See https://github.com/dotnet/arcade/pull/455

**Pull request closed without merging**